lib/hanami/commands/generate/secret.rb

```diff
@@ -0,0 +1,31 @@
+require 'hanami/commands/generate/abstract'
+require 'hanami/application_name'
+require 'securerandom'
+
+module Hanami
+  module Commands
+    class Generate
+      class Secret
+
+        def initialize(application_name)
+          @application_name = application_name
+        end
+
+        def start
```

Comment:
Just my personal opinion, the `start` method isn't quite relevant for this case, I think `execute!` is a better one.